frontend/src/Dashboard/CrossingDetailPage/CrossingDetailPage.js

```diff
@@ -1,19 +1,76 @@
 import React, { Component } from 'react';
+import { graphql, compose } from 'react-apollo';
+import gql from 'graphql-tag';
 import CrossingDetails from './CrossingDetails';
 import CrossingStatusHistory from './CrossingStatusHistory';
 
+
 class CrossingDetailPage extends Component {
   render() {
-    const crossingId = this.props.match.params.id;
+    if ( !this.props.CrossingByIdQuery ||
+          this.props.CrossingByIdQuery.loading ||
+          !this.props.CrossingHistoryQuery ||
+          this.props.CrossingHistoryQuery.loading ) {
```

Comment:
This feels like a long unruly list of ways our state can be loading. What if we captured that state this way...

```js
const isLoading = !this.props.CrossingByIdQuery || 
  this.props.CrossingByIdQuery.loading || 
  !this.props.CrossingHistoryQuery || 
  this.props.CrossingHistoryQuery.loading; 
```
and then use that `isLoading` variable in the if conditional.

Reply:
I really like this idea, I've used this pattern in a few places - created an issue https://github.com/cityofaustin/ctxfloods/issues/69